packages/js-sdk/src/ConstantFlowAgreementV1Helper.js

```diff
@@ -60,6 +60,7 @@ module.exports = class ConstantFlowAgreementV1Helper {
                 }
```

Comment:
We also need to add to the IDA helper as well?

Reply:
not now. as mentioned above, just trying to close the ticket and merge the code instead of letting it dangling around.